unsafe/src/main/java/org/apache/spark/unsafe/types/UTF8String.java

```diff
@@ -165,6 +165,38 @@ public UTF8String toLowerCase() {
     return UTF8String.fromString(toString().toLowerCase());
   }
 
+  public UTF8String initCap(final byte[] byteArr) {
+    String isoString = null;
+    try {
+      isoString = new String(byteArr, "US-ASCII");
```

Comment:
The new alphabets can be added according to this implementation, all the alphabets are in http://www.asciitable.com/ 
The customers can also make some enhancement based on this solution. Support this kind of alphabets are not only in this function.